### PR TITLE
Added cors package

### PR DESCRIPTION
**Pull request by Kajol-Kumari**
#73 [Improvement] Introduce CORS to the Notifier-Service express application

Files modified:
- package.json
- package-lock.json
- app.ts

### CODE REVIEW

File: Notifier-service/src/app.ts [hunk 1]
```diff
@@ -22,6 +23,7 @@ export class App {
 
     constructor(private container:AppContainer){
         this.app = express();
+        this.app.use(cors());
```
Comment (rajatkb) [inline Notifier-service/src/app.ts:26]: * CORS takes configuration information right. regarding which ip's to allow for incoming request. Maybe you can add configuration entry for that information in .env , and load that here with the cors.

Comment (Kajol-Kumari) [reply Notifier-service/src/app.ts]: @rajatkb Is it ok?

File: Notifier-service/.env [hunk 2]
```diff
@@ -3,4 +3,5 @@ MONGO_DB_PORT=27017
 MONGO_DB_NAME=Conference_Notify
 SERVER_PORT=3000
 LOG_FOLDER=logs
-LOG_LEVEL=debug
\ No newline at end of file
+LOG_LEVEL=debug
+ORIGIN=["http://localhost:3000", "http://localhost:3020"]
```
Comment (rajatkb) [inline Notifier-service/.env:7]: * Specify only one external host. Arrays wont work here

Comment (Kajol-Kumari) [reply Notifier-service/.env:7]: should i remove 3020 then?

Comment (rajatkb) [reply Notifier-service/.env]: * I have done the change it was minor.

**Pull request merged**